src/action.js

```diff
@@ -65,46 +57,19 @@ module.exports = async function run() {
       return logInfo(`${pkgName} is excluded, skipping.`)
     }
 
-    if (API_URL !== DEFAULT_API_URL &&
-      pkgName === 'github-action-merge-dependabot' &&
-      isMajorRelease(pr)) {
+    if (pkgName === 'github-action-merge-dependabot' && isMajorRelease(pr)) {
       logWarning(upgradeMessage)
       core.setFailed(upgradeMessage)
```

Comment:
is there a chance that this is redundant? I mean both logging and warning and failing the action with a message won't result in two identical logs?